src/util/tubemap.js

```diff
@@ -3240,6 +3248,8 @@ function filterObjectByAttribute(attribute, value) {
   return (item) => item[attribute] === value;
 }
 
+// draw clicks
+
```

Comment:
I don't think that's right?
```suggestion
```

Or you could explain how this draws "track rectangles", which I think are the straight non-bendy parts of the reads and paths.